MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -1913,10 +1913,14 @@ public GetViewTableDataResponse GetViewTableData (WebServiceLogin login, int? la
 		}
 
 		[WebMethod]
-		public GetViewWorkTableDataResponse GetViewWorkTableData (WebServiceLogin login, int? lane_id, string lane, int? host_id, string host, int? command_id, string command)
+		public GetViewWorkTableDataResponse GetViewWorkTableData (WebServiceLogin login, int? lane_id, string lane, int? host_id, string host, int? command_id, string command, int offset = 0, int limit = 250)
```

Comment:
Add a new method instead of changing the signature of an existing method, since otherwise it'll break any existing users.